frontend/src/services/fetchData.ts

```diff
@@ -83,11 +90,38 @@ export const fetchApplicationAdmins = async (): Promise<FamAppAdminGetResponse[]
  * @param applicationAdminId id to delete fam_user_role_assignment record.
  */
 export const deleteAndRefreshApplicationAdmin = async (
-    applicationAdminId: number,
+    applicationAdminId: number
 ): Promise<FamAppAdminGetResponse[]> => {
     await AdminMgmtApiService.applicationAdminApi.deleteApplicationAdmin(
         applicationAdminId
     );
     // When deletion is successful, refresh (fetch) for frontend state.
     return fetchApplicationAdmins();
 };
+
+export const fetchDelegatedAdmins = async (
+    applicationId: number | undefined
+): Promise<FamAccessControlPrivilegeGetResponse[]> => {
+    if (!applicationId && !FamLoginUserState.isAdminOfSelectedApplication()) {
```

Comment:
oh here I missed, it should be or "||", if no application is selected , or user is not admin of the selected app, return empty array

Reply:
sure, thanks!